tar/read.c

```diff
@@ -67,6 +67,8 @@ __FBSDID("$FreeBSD: src/usr.bin/tar/read.c,v 1.40 2008/08/21 06:41:14 kientzle E
 
 #include "bsdtar.h"
 #include "err.h"
+#include "archive_platform.h"
```

Comment:
I got this error on compiling - I saw trying to get the _O_BINARY value and the _setmode function.

12>C:\Users\User\Development\libarchive\libarchive\tar\read.c(242): warning C4013: '_setmode' undefined; assuming extern returning int
12>C:\Users\User\Development\libarchive\libarchive\tar\read.c(242): error C2065: '_O_BINARY' : undeclared identifier